src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -99,6 +110,39 @@ public static long nowInMicros()
         return TimeUnit.MILLISECONDS.toMicros(Clock.Global.currentTimeMillis());
     }
 
+    public TxnData coordinate(Txn txn)
+    {
+        try
+        {
+            Future<Result> future = node.coordinate(txn);
+            Result result = future.get(DatabaseDescriptor.getTransactionTimeout(TimeUnit.SECONDS), TimeUnit.SECONDS);
```

Comment:
nit: if someone does `500ms` this will be incorrect, we should do `TimeUnit.NANOSECONDS` to avoid this issue

Reply:
`ms` is the minimum unit, so I think `MILLISECONDS` should work